XPathParser.sln.DotSettings

```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
I currently don't use ReSharper and will probably use StyleCop.Analyzers in the future for ensuring code formatting consistency.

Would you mind removing the DotSettings file from the PR & adding it to the .gitignore file if required?